### DISCUSSION TITLE
I want to connect!!!

### DISCUSSION THREAD

**Discussion by JessicaX83** [Chia-Network/chia-blockchain#5894] [Support]
I am going to apologize from the start because I am beyond upset at the moment. I have spent over $4000 to build a PC and gather hardware for plotting and farming and I STILL cannot sync to the network even after installing 1.1.6. I have never in my life ever had this much trouble making a simple connection. This is beyond frustrating and I am nearly fed up with it.

**Comment by tiny0little** [comment]
inbound tcp/8444?

more here - https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444

**Comment by JessicaX83** [comment]
Yes I have tried everything under the sun. turning off upnp, turning on upnp, dmz, connect directly to modem without router or firewall. I even bought a new router, all to no avail. This is the 1st time I have ever had an issue like this.

**Reply by tiny0little** [reply]
what https://portchecker.co/ shows for tcp8444?

**Comment by CaptainCrypt** [comment]
You're not alone.  Having same issue. Was fine until May 19th and everything crapped the bed.  No idea why I can't sync.

**Comment by BrandtH22** [comment]
@JessicaX83 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
